instructure/lti_originality_report_example
# Assignment configuration

_Closes PLAT-2479_
Test Plan:
* Go to the assignment creation page and launch the tool from the similarity detection placement.
* Verify the tool launches and click on one of the checkboxes in the launched tool.
* Save the assignment and then edit it. Verify when the tool launches the same checkbox is still checked.
